qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -366,31 +421,64 @@ def _new_scrofulous_control(
     maximum_rabi_rate: float = 2.0 * np.pi,
     **kwargs
 ) -> DrivenControl:
-    """
-    SCROFULOUS control to compensate for pulse length errors.
+    r"""
+    Creates a Short Composite ROtation For Undoing Length Over and Under Shoot (SCROFULOUS) driven
```

Comment:
I think you don't need to "capitalize the spelled-out form of an acronym unless it's a proper noun" (https://docs.microsoft.com/en-us/style-guide/capitalization)

Reply:
TIL, thanks!